cfgov/tccp/jinja2/tccp/card.html

```diff
@@ -2,6 +2,7 @@
 
 {% from 'tccp/includes/data_published.html' import data_published %}
 {% from 'tccp/includes/fields.html' import apr, apr_range, currency, lower_first_letter, oxfordize %}
+{% from 'tccp/includes/speed_bump.html' import speed_bump %}
```

Comment:
I appreciate the reuse of the speed bump template here! Are you okay with the color changing to teal-10 here when it gets changed for the list view?